sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Microsoft.Azure.Management.Marketplace.csproj

```diff
@@ -6,9 +6,9 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Marketplace</PackageId>
     <Description>Microsoft Azure Marketplace Library</Description>
-    <Version>1.0.1</Version>
+    <Version>1.0.2</Version>
     <AssemblyName>Microsoft.Azure.Management.Marketplace</AssemblyName>
-    <PackageTags>management;marketplace;</PackageTags>
+    <PackageTags>management;marketplace;privatestore</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
         This is a public release of the Azure Marketplace SDK. Included with this release is the ability to create and manage private stores in Marketplace.
```

Comment:
Besides, please update the release notes. Thanks

Reply:
Release was updated